pom.xml

```diff
@@ -620,12 +629,12 @@
       <dependency>
         <groupId>org.reflections</groupId>
         <artifactId>reflections</artifactId>
-        <version>0.9.10</version>
+        <version>0.9.11</version>
       </dependency>
       <dependency>
         <groupId>org.rocksdb</groupId>
         <artifactId>rocksdbjni</artifactId>
-        <version>5.15.10</version>
+        <version>6.6.4</version>
```

Comment:
This dependency grows from 13MB to ~28MB according to this: https://mvnrepository.com/artifact/org.rocksdb/rocksdbjni

Just something to note

Additionally, upgrading a major version for rocksdb means there can be incompatibilities in the on-disk formats between versions and could result in backcompat issues for people using rocksdb. I think the highest version we can upgrade to is 5.18.3

Reply:
The new rocksdb version mainly adds support for arm and musl-libc. I changed the code to support a custom rocksdb version.